File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/PostalCode.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class PostalCode 
{
    public long Id { get; set; }
    public bool Active { get; set; }
    public bool Valid { get; set; }
    public double Latitude { get; set; }
    public double Longitude { get; set; }
    public string LocationDescription { get; set; }
    public string Code { get; set; }
    public string Plus4 { get; set; }
    public string StateCode { get; set; }
    public string City { get; set; }
    public string VarTimeZone { get; set; }
    public int UtcOffset { get; set; }
    public List<Region> Regions { get; set; }
    public bool Eliminated { get; set; }
    public bool SearchPoint { get; set; }
    public DateTime LastOfferImport { get; set; }
    public double TruncatedLatitude { get; set; }
    public double TruncatedLongitude { get; set; }
}

File: server-scala-play-server/app/model/PostalCode.scala
package model

import play.api.libs.json._
import java.time.OffsetDateTime
import scala.collection.immutable.Set

/**
  * Represents the Swagger definition for PostalCode.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class PostalCode(
  id: Option[Long],
  active: Option[Boolean],
  valid: Option[Boolean],
  latitude: Option[Double],
  longitude: Option[Double],
  locationDescription: Option[String],
  code: Option[String],
  plus4: Option[String],
  stateCode: Option[String],
  city: Option[String],
  timeZone: Option[String],
  utcOffset: Option[Int],
  regions: Option[Set[Region]],
  eliminated: Option[Boolean],
  searchPoint: Option[Boolean],
  lastOfferImport: Option[OffsetDateTime],
  truncatedLatitude: Option[Double],
  truncatedLongitude: Option[Double]
)

object PostalCode {
  implicit lazy val postalCodeJsonFormat: Format[PostalCode] = Json.format[PostalCode]
}

